lib/local-env-helper.js

```diff
@@ -0,0 +1,40 @@
+/**
+ *  This include should required by any command-line utility that requires AWS
+ *  credentials and environment-specific ENV variables. By including this file,
+ *  the process.argv will be inspected for the following params:
+ *
+ *   - `--profile`: AWS profile to use
+ *   - `--envfile`: Path to an environment env file containing deployable ENV
+ *                  varialbes (e.g. config/qa.env)
+ *
+ *  If both parameters are given, any aws-sdk calls made subsequently will be
```

Comment:
If we require this file we're saying that `profile` and `envfile` are required, parameters right? This kinda sounds to me like they're optional.